services/src/functional-tests/kilda.properties.example

```diff
@@ -22,6 +22,7 @@ elasticsearch.endpoint = http://localhost:9200
 elasticsearch.username = username
 elasticsearch.password = password
 
+environment.naming.prefix=
```

Comment:
Please, provide an example value for this property.

Reply:
Ok. So I think we can leave a small comment that will explain that for virtual env it should be empty, for staging it should be "staging".

Reply:
all staging info is 'internal' for us, so should not be pushed to public